docs/entityoperator.md

```diff
@@ -0,0 +1,315 @@
+
+
+### Summary of How Applications Use KafkaTopic & KafkaUser CRs
+
+| Action    | Operator Responsible |
+| -------- | ------- |
+| Developer creates a `KafkaTopic` CR  | Topic Operator creates & syncs the topic in Kafka    |
+| Developer creates a KafkaUser CR | User Operator creates the user & credentials  |
+| Application retrieves credentials from Kubernetes Secrets	| Application mounts the secrets for authentication |
+| Application connects to Kafka using these credentials | Producer/Consumer communicates with Kafka  |
+
+
+## Impact of Central Cluster Failure on Kafka Clients in a Stretch Cluster
+
+In stretch Kafka deployment, where
+
+✅ Kafka brokers and controllers are spread across multiple Kubernetes clusters.<br>
+✅ The central cluster hosts all Kafka CRs, including Kafka, KafkaNodePool, KafkaUser, and KafkaTopic.<br>
+✅ The Entity Operator (managing users & topics) runs in the central cluster.
+
+
+The failure of the central Kubernetes cluster will render the Entity Operator unavailable. This has the following implications for Kafka clients
+
+#### Authentication
+
+- Kafka brokers in the surviving member clusters rely on the configured authentication mechanisms and the presence of valid credentials for client authentication.
+- If TLS certificates secrets are not replicated across all clusters, new client deployments and credential updates will fail. However, existing clients with valid TLS certificates will continue functioning until their certificates expire or require rotation. The duration for which they can operate depends entirely on the expiration date set when the certificates were issued.
+- If SCRAM credentials have been successfully replicated across the Kafka brokers, existing clients should be able to continue authenticating, even if the Central cluster is down. The issue is with new client deployments or credential updates.
+- Existing client connections that were authenticated before the central cluster failure might remain active for a period, but their continued operation depends on multiple factors. If a Kafka broker restarts, clients may need to re-authenticate, which could fail if they rely on new credentials from an unavailable Entity Operator. Additionally, the configured `session.timeout.ms` and Kafka’s reauthentication behavior may determine how long clients remain connected before being disconnected.
+- Crucially, the management of credentials (e.g., rotation) through the User Operator will be unavailable.
```

Comment:
I don't this this is presenting a clear picture and there are a number of sentences with `might` which we shoudl change to a more assertive sentence.

IMO:
- administrators will not be able to view create, update, or delete Kafka users and their credentials via Kubernetes custom resources.
- - administrators will not be able to view create, update, or delete Kafka topics and their configuration via Kubernetes custom resources.
- existing users, topics and client applications using those credentials will continue to work as usual with no disruption as long as the credentials remain valid.

Reply:
if we really need to call out a plan B, we should provide steps on how can someone setup entity operator on a different cluster using custom resource definitions from a GitOps branch.